src/rebar3.erl

```diff
@@ -320,7 +322,6 @@ ensure_running(App, Caller) ->
     end.
 
 state_from_global_config(Config, GlobalConfigFile) ->
-    rebar_utils:set_httpc_options(),
```

Comment:
The distinction would be that we allow people to set their proxy configuration in the global config files; if they're in the config file but not the environment, the proxy should still work. You're entirely right with `get_http_vars` so this should keep working.